### ISSUE TITLE
Add recipe for AspNetCore.SpaTemplates

### ISSUE THREAD

**Issue by johnpankowicz** [Microsoft/vscode-recipes#40]
If I create a project using "dotnet new angular", I can debug both  the C# and Typescript code in Visual Studio 2017. But within Visual  Studio Code, I can only debug the C#. When I try to place a breakpoint  on any Typescript instruction, it says: "No symbols have been loaded for  this document".
Please see the following Stackoverflow post for more info: https://stackoverflow.com/questions/47598215/cant-debug-typescript-in-vscode-on-app-created-with-dotnet-new-angular

**Comment by auchenberg** [comment]
@johnpankowicz You'll also need to make a debug configuration for your Chrome debugger. See the instructions here https://github.com/Microsoft/vscode-recipes/tree/master/Next-js#configure-launchjson-file, and ignore Nodejs config.

To debug both .net core and Chrome at the same time, you can make a `compound` configuration, which is explained there to.

It would end up being a config like
```json
   {
   "version": "0.2.0",
   "configurations": [
        {
            "name": ".NET Core Launch (web)",
            "type": "coreclr",
            "request": "launch",
            "preLaunchTask": "build",
            // If you have changed target frameworks, make sure to update the program path.
            "program": "${workspaceFolder}/bin/Debug/netcoreapp2.0/spa.dll",
            "args": [],
            "cwd": "${workspaceFolder}",
            "stopAtEntry": false,
            "internalConsoleOptions": "openOnSessionStart",
            "launchBrowser": {
                "enabled": true,
                "args": "${auto-detect-url}",
                "windows": {
                    "command": "cmd.exe",
                    "args": "/C start ${auto-detect-url}"
                },
                "osx": {
                    "command": "open"
                },
                "linux": {
                    "command": "xdg-open"
                }
            },
            "env": {
                "ASPNETCORE_ENVIRONMENT": "Development"
            },
            "sourceFileMap": {
                "/Views": "${workspaceFolder}/Views"
            }
        },
        {
            "name": ".NET Core Attach",
            "type": "coreclr",
            "request": "attach",
            "processId": "${command:pickProcess}"
        },
        {
            "type": "chrome",
            "request": "launch",
            "name": "Chrome",
            "url": "http://localhost:3000",
            "webRoot": "${workspaceRoot}"
        }
    ],
    "compounds": [
        {
            "name": "Full stack",
            "configurations": [".NET Core Launch (web)", "Chrome"]
        }
    ]
}
```
Let me know how things work.

**Comment by johnpankowicz** [comment]
@auchenberg Thanks a lot. That worked. I made just a couple minor changes to your sample launch.json. In the Chrome config, I have:  "webRoot": "${workspaceRoot}/wwwroot". And I changed the "url" port to "5000" to match what the C# for Visual Studio Code extension uses. 

One issue is the app is launched twice, once in my open default browser and again in Chrome. I can avoid the first by setting "enabled" ...  to "false" for LaunchBrowser in the ".NET Core Launch (web)" config. So I may need to create a second version of ".NET Core Launch (web)" for when I am using the compound configuration.

**Comment by auchenberg** [comment]
@johnpankowicz Great, glad to hear you got it working!

I'll let this issue stay open, so we can add writing a dotnet core recipe to our backlog. In case you have time a PR with your working configuration would be much appreciated!

**Comment by johnpankowicz** [comment]
I added the PR right after you suggested a week ago. How often are these looked at?

**Comment by auchenberg** [comment]
Looks good, thanks for the PR @johnpankowicz!

**Comment by johnpankowicz** [comment]
@auchenberg - I have a couple issues in my README.md for the PR.
(1) A code block didn't get formatted. (In MY viewer it looked ok) I think a missing indent on the first line is causing the problem.
(2) My images are not showing. I copied how the Electron recipe did them , since those are also in the same folder as the Readme.md. EG: Electron uses "   ![configure_launch](configure_launch.png)" I used "![Typescript breakpoint](Typescript breakpoint.png)". Could the lack of an indent or a space in the filename have something to do with it? Again, it appears ok in my .md viewer.
